bfcfg

```diff
@@ -648,29 +655,157 @@ sys_cfg_one_byte()
   fi
 }
 
+#
+# Dump a multi-byte value at the specified offset in a file.
+#
+sys_cfg_multi_byte()
+{
+  local tmp_file=$1
+  local name=$2
+  local offset=$3
+  local num_bytes=$4
+  local value=$5
+
+  if [ ${dump_mode} -eq 0 ]; then
+    return
+  fi
+
+  value=0
+  if [ -e ${tmp_file} ]; then
+    value=$(hexdump -s "${offset}" -n "${num_bytes}" -e '"0x%X" "\n"' "${tmp_file}")
```

Comment:
Is there any case that the output could be over one line thus the output might be not expected?

Reply:
Not sure I fully understand - do you mean if we tried to dump too many bytes that it would look bad (e.g. `sys: VAR=0x1234678901234567809012325366708...`?

`sys_cfg_multi_byte` is meant to be used as private function called by other functions like `sys_cfg` so it doesn't need to handle a variable input from running `bfcfg`, for example.

But we could enforce a max `num_bytes` size in the function or split the output into multiple lines if `num_bytes` was larger than a set threshold.

Not sure it's necessary now though since this is only being used to dump 2-4 bytes (meant mostly for UINT16 and UINT32 sys config vars). What do you think?